Keycloak/legos/keycloak_get_audit_report/keycloak_get_audit_report.py

```diff
@@ -48,6 +58,25 @@ def keycloak_get_audit_report(handle) -> List:
         return events if events else []
 
     except Exception as e:
-        raise e
+        try:
```

Comment:
Keep it simple, assume we are using request based keycloak connector

Reply:
Done